src/common/collections.rs

```diff
@@ -51,3 +57,61 @@ pub async fn do_get_collection_cluster(
     let collection = toc.get_collection(name).await?;
     Ok(collection.cluster_info(toc.this_peer_id).await?)
 }
+
+pub async fn do_update_collection_cluster(
+    toc: &TableOfContent,
+    collection_name: String,
+    operation: ClusterOperations,
+    dispatcher: &Dispatcher,
+    wait_timeout: Option<Duration>,
+) -> Result<bool, StorageError> {
+    if dispatcher.consensus_state().is_none() {
+        return Err(StorageError::BadRequest {
+            description: "Distributed mode disabled".to_string(),
+        });
+    }
+    let consensus_state = dispatcher.consensus_state().unwrap();
```

Comment:
This probably looks cleaner and no unwraps there :smile: 
```rust
let consensus_state = dispatcher.consensus_state() {
    Some(state) => state,
    None => return Err(..),
}
```

Reply:
Thanks, it does look better.
I will remember this trick :+1: